pulp_smart_proxy/app/views.py

```diff
@@ -0,0 +1,69 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+from drf_spectacular.utils import extend_schema
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.apps import pulp_plugin_configs, get_plugin_config
+
+
+class FeaturesView(APIView):
+    """
+    Returns features of the smart_proxy
+    """
+
+    @extend_schema(
+        summary="Inspect features",
+        operation_id="features_read",
+    )
+    def get(self, request):
+        data = ["pulpcore"]
+        return Response(data)
+
+
+class FeaturesV2View(APIView):
+    """
+    Returns features of the smart_proxy in v2 format
+    """
+
+    @extend_schema(
+        summary="Inspect features",
+        operation_id="featuresv2_read",
+    )
+    def get(self, request):
+        host = settings.CONTENT_ORIGIN or request.get_host()
+        capabilities = [app.label for app in pulp_plugin_configs()]
+        data = {
+            "pulpcore": {
+                "http_enabled": False,
+                "https_enabled": True,
+                "settings": {
+                    "pulp_url": host,
+                    "mirror": settings.SMART_PROXY_MIRROR,
+                    "content_app_url": urljoin(host, settings.CONTENT_PATH_PREFIX),
```

Comment:
I think this should be `settings.CONTENT_ORIGIN`. AFAIK that's the actual content APP, which can run on a different URL and port.